src/backend.rs

```diff
@@ -724,6 +724,40 @@ impl SharedBackend {
     pub fn flush_cache_to(&self, cache_path: &Path) {
         self.cache.0.flush_to(cache_path);
     }
+
+    /// Returns the DB
+    pub fn data(&self) -> DatabaseResult<Arc<MemDb>> {
```

Comment:
looks like we don't need the result on any of these functions

Reply:
Fixed